### ISSUE TITLE
Adding Collab Cycle Request, feedback and finding tickets to milestone

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#63011]
## User Story
As a Governance team member, I want to have milestones created in Github for all historical Collaboration Cycle request initiatives so that I can use this data in our modified Collaboration Cycle dashboard in Domo. 

Assignee: @it-harrison 
Peer Reviewer: @briandeconinck 

## Description
In ticket #62561, Governance team member @it-harrison documented a solution for creating milestones in GH and ensuring all historical collab cycle requests, design intent feedback tickets, midpoint review feedback tickets and Staging Review findings tickets are added to the appropriate milestone.

The purpose of this ticket is to implement the solution.

## Tasks
- [x] Execute the [solution](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2736193587/62561+-+Automation+Opportunities)
- [x] Validate that uniquely named milestones have been created
- [x] Validate that the CC Request, DI and MPR feedback tickets, and SR findings tickets have been added to the appropriate milestone

## Peer Review
_To be completed by peer reviewer_
- [ ] User story and acceptance criteria are met

## Acceptance Criteria
- [ ] All Collaboration Cycle Request tickets in the va.gov-team repo have been added to a new milestone
- [ ] All Design Intent feedback tickets that are child tickets of the Collaboration Cycle Request ticket epic have been added to the appropriate milestone
- [ ] All Midpoint Review feedback tickets that are child tickets of the Collaboration Cycle Request ticket epic have been added to the appropriate milestone
- [ ] All Staging Review finding tickets that are child tickets of the Collaboration Cycle Request ticket epic have been added to the appropriate milestone

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by shiragoodman** [comment]
@it-harrison  - on our team channel yesterday, [Naomi asked](https://dsva.slack.com/archives/GNF5GSD9N/p1692133334650039) if we had metrics around average time from Design Intent to Staging Review. I _think_ the only way we might be able to calculate this with Milestones is if we pull in the "Completed" tickets to the milestone. Could you think of another way we could measure this? If pulling in the "Completed" tickets is the only way we could capture the date that a review took place, then I think we'll want to modify this ticket to pull in those tickets as well. Let me know what you think!

**Comment by it-harrison** [comment]
@shiragoodman I think that would be the easiest way, especially if Naomi wants this in Domo. It should also be possible to parse the text of the Collab Cycle Ticket which might be more accurate in some cases (if the Completed Ticket isn't closed on the day of the meeting) but this method would be generated by a script and not in Domo.

**Comment by shiragoodman** [comment]
I think a day or 2 difference in the completed tickets wouldn't impact the average time between touchpoints too much. Parsing the data seems like a lot of work.

Would you be able to add the "completed" tickets to the scope of this ticket?

**Comment by it-harrison** [comment]
@shiragoodman yes, the "completed" tickets will be added for free because they are part of the epic

**Comment by it-harrison** [comment]
@briandeconinck this is ready for review - all relevant tickets should have a milestone.

**Comment by shiragoodman** [comment]
@it-harrison I found a couple tickets that didn't get milestones created. 
https://github.com/department-of-veterans-affairs/va.gov-team/issues/54257
https://github.com/department-of-veterans-affairs/va.gov-team/issues/55260
Looks like Matt had removed the `CC-request` label back in July. I'll let him know that this label is critical and not to remove it in the future.

**Comment by briandeconinck** [comment]
Found a few more that didn't get milestones. Not sure if these were also missing `CC-request` and Shira added that label, or if they got skipped for some other reason.

- #50380
- #48859
- #54257

I'm pretty sure these are the only ones that didn't get milestones, plus the two Shira already flagged. All of the related touchpoint tickets that I picked at random looked like they were correctly added to the milestones.
